sdk/python/kfp/compiler/compiler.py

```diff
@@ -42,16 +42,48 @@
 
 _GroupOrTask = Union[tasks_group.TasksGroup, pipeline_task.PipelineTask]
 
+CompileSignature = Callable[..., None]
+
+
+def _alias_deprecated_params(
```

Comment:
Nice util! Wondering if it's beneficial to put in utils folder, so other parts could also use this.

Reply:
Good idea -- will do!